src/App.js

```diff
@@ -1,24 +1,18 @@
-import logo from './logo.svg';
 import './App.css';
+import Homepage from './pages/Homepage/Homepage';
+import { Route, Routes } from 'react-router-dom';
+import Header from './components/Header/Header';
+import Footer from './components/Footer/Footer';
 
 function App() {
   return (
-    <div className="App">
-      <header className="App-header">
-        <img src={logo} className="App-logo" alt="logo" />
-        <p>
-          Edit <code>src/App.js</code> and save to reload.
-        </p>
-        <a
-          className="App-link"
-          href="https://reactjs.org"
-          target="_blank"
-          rel="noopener noreferrer"
-        >
-          Learn React
-        </a>
-      </header>
-    </div>
+    <>
+      <Header />
+      <Routes>
```

Comment:
Instead of adding routes in the App.js file, you could move it to a separate file say `WebsiteRoutes.js` and then add the component here. This way, in the future, if the number of routes increases your code will look cleaner, readable and decomposed.